express-zod-api/tests/__snapshots__/documentation.spec.ts.snap

```diff
@@ -1336,17 +1321,29 @@ paths:
 components:
   schemas:
     Schema1:
-      type: array
-      items:
-        type: object
-        properties:
-          name:
-            type: string
-          subcategories:
+      type: object
+      properties:
+        name:
+          type: string
+        subcategories:
+          type: array
+          items:
             $ref: "#/components/schemas/Schema1"
-        required:
-          - name
-          - subcategories
+      required:
+        - name
+        - subcategories
+    Schema2:
```

Comment:
these 2 identical schemas could possibly be deduplicated when using a higher order registry

Reply:
made todo, will address in another PR